templates/spec/controllers/checkouts_controller_spec.rb

```diff
@@ -109,85 +109,4 @@
       end
     end
   end
-
-  context '#registration' do
-    it 'does not check registration' do
-      expect(controller).not_to receive(:check_registration)
-      get :registration
-    end
```

Comment:
I no longer included this spec in the CheckoutSessionsController spec because it is no applicable.